src/app/proof-of-liabilities/get-user-liabilities.ts

```diff
@@ -0,0 +1,18 @@
+import { LedgerService } from "@services/ledger"
+import { WalletsRepository } from "@services/mongoose"
+
+export const getUserLiabilities = async () => {
+  const wallets = await WalletsRepository().listAll("BTC")
+  if (wallets instanceof Error) return wallets
+  const liabilities = wallets.map((wallet) => walletToLiability(wallet))
+  return liabilities
+}
+
+const walletToLiability = async (wallet: Wallet) => {
```

Comment:
this is just a `getBalance` function. we should already have this as part of the API for wallet